src/api/middleware/errorMiddleware.ts

```diff
@@ -4,6 +4,8 @@ interface ConfluenceResponse extends Omit<Response, 'data'> {
   data: () => {
     message: string
   }
+  responseData?: unknown
+  errors?: unknown[]
```

Comment:
I don't see the `errors` field on the `Response` type:

https://github.com/tulios/mappersmith/blob/18abaca9b987b0d71301b9582b490b256edfc42e/typings/index.d.ts#L62-L74

There is an `error` field, but no `errors`.

Reply:
In fact, I think this type can be replaced with just referencing the `Response` type directly, since this type doesn't actually do anything meaningful. The `data` method is already a generic that allows you to define the return type.